src/prerender.ts

```diff
@@ -148,6 +148,10 @@ async function prerender({
             return
           }
         }
+        const routeResult = await route(url, allPageIds, contextProps)
+
+        Object.assign(contextProps, { url }, routeResult?.contextPropsAddendum)
+
```

Comment:
I believe we don't need this anymore: there is now `contextProps.urlFull`, `contextProps.urlPathname`, and `cotnextProps.urlParsed`.